README.md

```diff
@@ -1,6 +1,8 @@
 # WakaTime CLI
 
-![Tests](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Create%20Release/release?label=%20tests) ![Build](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Build%20and%20upload%20release%20assets) [![Coverage Status](https://coveralls.io/repos/github/wakatime/wakatime-cli/badge.svg?branch=release)](https://coveralls.io/github/wakatime/wakatime-cli?branch=release)
+![Tests](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Tests/develop?label=tests)
+![Build](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Build%20and%20upload%20release%20assets)
```

Comment:
Since we unified workflows it has changed. You need to rebase and get the new one.

Reply:
Here's the list of current workflows we can choose from:

https://api.github.com/repos/wakatime/wakatime-cli/actions/workflows

Reply:
We only have one workflow now `Tests` so we should remove the Build badge.